gatsby-node.js

```diff
@@ -60,10 +79,20 @@ exports.sourceNodes = async ({ actions, createNodeId, createContentDigest }, opt
   for (const _path of paths) {
     let githubPath = path.join(root, _path.replace(process.cwd(), ''))
     if (githubPath.charAt(0) === path.sep) githubPath = githubPath.substr(1)
+    if (prefix && githubPath.startsWith(prefix)) {
+      githubPath = githubPath.slice(prefix.length + 1)
```

Comment:
This would remove one extra character (so the prefix entirely plus the next character after). Is this the intent

Reply:
Yeah it gets rid of the /. If I don't get rid of it the graph ql query fails.